### PR TITLE
upgrade buildnet deps

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]
LGTM

File: packages/sc-deployer/package-lock.json
Comment (Thykof) [inline packages/sc-deployer/package-lock.json:1]: why do you delete this file?

Comment (peterjah) [reply packages/sc-deployer/package-lock.json:1]: it seems not used !

Comment (peterjah) [reply packages/sc-deployer/package-lock.json:1]: yes!

Comment (peterjah) [reply packages/sc-deployer/package-lock.json:1]: all packages use the same package-lock at the repo root ...

Comment (peterjah) [reply packages/sc-deployer/package-lock.json:1]: you can see that there is no package-lock for the other package (initializer)

**Pull request merged**